services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/northbound/flows/FlowPingSpec.groovy

```diff
@@ -224,7 +224,30 @@ class FlowPingSpec extends BaseSpecification {
     }
 
     def getOfSwitchCombinations() {
-        return [topology.activeSwitches, topology.activeSwitches].combinations().findAll { src, dst -> src != dst }
-                .unique { it.ofVersion.sort() }
+        def nbSwitches = northboundService.getActiveSwitches()
```

Comment:
Wouldn't it be better to use nonCentecSwitches() instead of getActiveSwitches()? Looks like Centecs are filtered out anyway.

Reply:
The problem is that nonCentecSwitches returns list of Switch objects (topology.yaml format), while I need SwitchInfoData here (northbound format).